optaweb-employee-rostering-frontend/src/ui/pages/employee/EmployeesPage.tsx

```diff
@@ -153,6 +166,18 @@ export class EmployeesPage extends DataTable<Employee, Props> {
         value={data.skillProficiencySet ? data.skillProficiencySet : []}
         onChange={selected => setProperty('skillProficiencySet', selected)}
       />,
+      <TextInput
+        key={3}
+        name="shortId"
+        defaultValue={data.shortId}
+        aria-label="shortId"
+        onChange={value => setProperty('shortId', value)}
+      />,
+      <ColorPicker
+        key={4}
```

Comment:
This is not a dynamic list of elements. I think the key is not needed.

Reply:
You still need keys for a static list of elements; For instance, if you add a button to the actions prop of a modal without a key, you get the missing key warning in the web console.